target/linux/ramips/image/mt7621.mk

```diff
@@ -336,6 +350,23 @@ define Device/iodata_wn-ax1167gr
 endef
 TARGET_DEVICES += iodata_wn-ax1167gr
 
+define Device/iodata_wn-dx1167r
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  UIMAGE_MAGIC := 0x434f4d43
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 51200k
+  DEVICE_VENDOR := I-O DATA
+  DEVICE_MODEL := WN-DX1167R
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | custom-initramfs-uimage 3.10(XIK.1)b10 | \
+	iodata-mstc-header
+  KERNEL_INITRAMFS_SUFFIX := -factory.bin
```

Comment:
Is this really necessary?

Reply:
Indeed, this is not required.

Reply:
I thought it is for indicating factory image, isn't it? I was planning to add it in ipTIME devices too...

Reply:
In this devices, the initramfs image can be used as a factory image directly, so I think this is not required. However, as you say, it maybe indicates more clear.

Reply:
Yes, but isn't it obvious that the initramfs image is used for factory flashing? Or, asked differently, how would you label the "other" initramfs images then?

Reply:
We are currently planning to make naming of images a little more systematic:
https://github.com/openwrt/openwrt/pull/2727

So, I'm not against adding -factory.bin in general, but if we start to use it, it should be done with some system. Consequently, I mainly ask myself how different the situation here is from other devices that use standard -kernel.bin ...

Reply:
Ah, okay, now I see your point.

Reply:
Despite, if we reconsider, it's easy to change and won't have any compatibility issues.